### PR TITLE
Adding ec2 instance launch

### PR DESCRIPTION
**Pull request by kjvenky**

### CODE REVIEW

**Comment by DinisCruz** [comment]
Hi @khalavak can you review this?

What do you think?

File: cloudformation/EC2WithSecurityGroup/ec2.yaml [hunk 1]
```diff
@@ -0,0 +1,338 @@
+AWSTemplateFormatVersion: '2010-09-09'
```
Comment (ajohnstone) [inline cloudformation/EC2WithSecurityGroup/ec2.yaml]: Use autoscaling not single instances.

File: cloudformation/EC2WithSecurityGroup/ec2.yaml [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+Metadata: 
+  License: Apache-2.0
```
Comment (ajohnstone) [inline cloudformation/EC2WithSecurityGroup/ec2.yaml]: Configure the instance automatically with the launch config

File: cloudformation/EC2WithSecurityGroup/ec2.yaml [hunk 3]
```diff
@@ -0,0 +1,338 @@
...
+Description: 'AWS CloudFormation Sample Template EC2InstanceWithSecurityGroupSample:
+  Create an Amazon EC2 instance running the Amazon Linux AMI. The AMI is chosen based
```
Comment (ajohnstone) [inline cloudformation/EC2WithSecurityGroup/ec2.yaml]: Are you defaulting to public ssh?

**Comment by DinisCruz** [comment]
I'm going to close this one since @kjvenky is not part of the team anymore

**Pull request closed without merging**